v8/hide_input_cells/hide_input_cells.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
+from nikola.plugin_categories import ConfigPlugin
+
+
+class HideInputCells(ConfigPlugin):
+    def set_site(self, site):
+        site.template_hooks['extra_head'].append(
+            """
+            <!--Code inserted by HideInputCells -->
+            <style type="text/css">
+              div.input {
```

Comment:
Those selectors feel too wide, are you sure you can’t use something that would only work on Jupyter notebooks and not all potential `div.input`s and `div.prompt`s out there?